README.md

```diff
@@ -1,2 +1,55 @@
 # Shift_Resolve
-Shift catalog of Operators for Resolve
+
+## Description
+
+Shift catalog of Operators for Davinci Resolve.
+This catalog include multiple Shift operators to manage your Resolve project; the media pool elements like folders, videos and images; the timelines and the timeline items; the takes; and much more.
+
+## Installation
+
+To add this catalog to Shift, you can open the Catalog Manager `File > Catalog Manager` and add the catalog like and user Catalog. Just click on the `Add Catalog` button and select the shift_resolve.py file.
+
+The catalog can be added through the environment too. For that add the path to the directory where the shift_resolve.py file is stored to the environment variable `SHIFT_CATALOG_PATH` from your environment before start Shift.
+
+Take in consideration that the catalog can be added and used to create workflows in any instance of Shift. However, to be able to execute the operators from the catalog, Shift have to be open inside Davinci Resolve and with the Resolve Python API available. To set up the Python Interpreter and the Python API requirements from Danvici Resolve is recommended to check the official documentation from your Davinci Resolve version. You can also find a detailed explanation of how to setup Shift in Davinci Resolve in the [Resolve Shift Documentation](https://inbibo.co.uk/docs/shift/integration_resources/software/resolve).
+
+## Dependencies
+
+| **Dependency**                                                              | **Version** |
+|-----------------------------------------------------------------------------|-------------|
+| [Shift](https://inbibo.co.uk/shift)                      | \>= 1.0.0   |
```

Comment:
Can you do it so the version is aligned in the column in the raw version?
That way is not shifted if the user opens the readme in a simple text editor.

Reply:
I tried, but Pycharm move everything automatically. Let me do it outside Pycharm and check that display correct

Reply:
done